VATSpy.dat

```diff
@@ -18560,7 +18560,13 @@ ZYSH|Shenyang|ZYSH|
 ZYTX|Shenyang ACC - Shenyang|ZYTX|
 ZYHB|Harbin ACC - Shenyang|ZYHB|
 ZYTL|Dalian ACC - Shenyang|ZYTL|
-ZMUB|Ulaan Baatar|ZMUB|
+ZMUB|Ulaanbaatar|ZMUB|

```

Comment:
Please change to this:
ZMUB|Ulaanbaatar||ZMUB

Reply:
Done. But what is the reason for that?

Reply:
I see. Would you please tell me the meaning of those spaces? And should I change all others of our airspace?